gallery/plot_datapoints.py

```diff
@@ -105,8 +151,8 @@ class PennFudanDataset(torch.utils.data.Dataset):
     def __getitem__(self, item):
         ...
 
-        target["boxes"] = datapoints.BoundingBoxes(
-            boxes,
+        target["bboxes"] = datapoints.BoundingBoxes(
+            bboxes,
```

Comment:
I would prefer we stick to the terms given in the tutorial this example is based on: https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html

```suggestion
        target["boxes"] = datapoints.BoundingBoxes(
            boxes,
```

Reply:
I'm not sure, I think we want to be consistent with the names used in this tutorial. What's being used in another tutorial is lower in priority when it comes to consistency

Reply:
OK, I'll move this section into another file, so let's revisit there. I think we should try find a better way to illustrate all this anyway because right now that section isn't even executable